lib/src/result/result.dart

```diff
@@ -64,7 +64,7 @@ abstract class Result<T> {
     try {
       return ValueResult<T>(computation());
     } catch (e, s) {
-      return ErrorResult(e, s);
+      return ErrorResult(e as Object, s);
```

Comment:
https://github.com/dart-lang/sdk/issues/41074

Looks like `on Object catch` might be the preference.

Reply:
done